### PR TITLE
Add Settings for Oscilloscope instrument #238

### PR DESCRIPTION
**Pull request by fragm3**
Fixes: #238 
**Changes:**
- [x] Add settings for Oscilloscope instrument
- [x] Created Reusable style, component
- [x] Created constants for options for select

**Screenshot**
Screenshot 2019-04-17 at 3 51 30 PM
Screenshot 2019-04-17 at 4 01 48 PM

**Note:** Slider for Timebase is not working

### CODE REVIEW

**Comment by fragm3** [comment]
@CloudyPadmal @mariobehling please review and share some feedback if you want any changes

File: src/screen/Oscilloscope/components/Settings.styles.js [hunk 1]
```diff
@@ -5,13 +5,23 @@ export const SettingsContainer = styled.div`
 	height: 100%;
```
Comment (AakashMallik) [inline src/screen/Oscilloscope/components/Settings.styles.js]: Change the code of SettingsContainer to this 
```
export const SettingsContainer = styled.div`
	width: 100%;
	height: calc(100% - 16px);
	display: flex;
	flex-direction: column;
	margin: 8px 0px 0px 16px;
`;
```

File: src/screen/Oscilloscope/components/Settings.styles.js [hunk 2]
```diff
@@ -5,13 +5,23 @@ export const SettingsContainer = styled.div`
...
-`;
\ No newline at end of file
+`;
+
+export const OptionsRowWrapper = styled.div`
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin: 8px 0px;
+	min-windth: 80%;
```
Comment (AakashMallik) [inline src/screen/Oscilloscope/components/Settings.styles.js]: min-width **typo**

File: src/screen/Oscilloscope/components/Settings.styles.js (hunk 1 above)
Comment (AakashMallik) [reply src/screen/Oscilloscope/components/Settings.styles.js]: This will prevent the scrollbar overlap

Comment (fragm3) [reply src/screen/Oscilloscope/components/Settings.styles.js]: Thanks for this 🎉

**Comment by fragm3** [comment]
@AakashMallik made the changes 🚀

**Pull request merged**